tests/test_asyncio.py

```diff
@@ -14,7 +14,7 @@ def test_time_freeze_coroutine():
     async def frozen_coroutine():
         assert datetime.date.today() == datetime.date(1970, 1, 1)
 
-    asyncio.get_event_loop().run_until_complete(frozen_coroutine())
+    asyncio.new_event_loop().run_until_complete(frozen_coroutine())
 
 
 def test_time_freeze_async_def():
```

Comment:
this test can just be removed because it's the same as the above test_time_freeze_coroutine